app/templates/index.html

```diff
@@ -83,9 +83,10 @@
                     </header>
                     <div class="card-image has-text-centered">
                         {% if show_video %}
```

Comment:
Should this be conditional if the camera is enabled and/or connected?

Reply:
If you want. Personally, I wouldn't think so. There's already a status indicator in the header row. I think a non-working video is a reasonable expectation for an offline camera.